### DISCUSSION TITLE
About this project

### DISCUSSION THREAD

**Discussion by kevinxxq** [AllenDang/giu#292] [General]
Hi All,
I have some questions about this project:
From the introduction, "A rapid cross-platform GUI framework for Go based on Dear ImGui and the great Go binding imgui-go." 
1). Is this a wrapper or extension of Dear IMGUI(https://github.com/ocornut/imgui)? Can I use all of Dear IMGUI?
2). Is this a wrapper or extension of wrapper (https://github.com/inkyblackness/imgui-go)? Can I use all of imgui-go?

Thank you.

**Comment by gucio321** [comment]
@kevinxxq yah it is a wrapper of wrapper of Dear ImGUI, so:
here is a small preview graphics I've just created
![image](https://user-images.githubusercontent.com/73652197/156941045-a0d730ad-7157-495e-9c87-042f7482b65d.png)
